libs/build_jasper.sh

```diff
@@ -56,13 +56,18 @@ else
     useCmake=NO
 fi
 
+# Load jpeg module if created by hpc-stack; requires setting
+# MAKE_POLICY_DEFAULT_CMP0074 to new below so that JPEG_ROOT is searched
+module try-load jpeg
```

Comment:
This has to be up top in the `if $MODULES` block.

Reply:
It hopefully is now.